main.go

```diff
@@ -8,10 +8,11 @@ import (
 	"time"
 
 	"github.com/Masterminds/semver"
-	"github.com/google/go-github/github"
+	"github.com/google/go-github/github"	
```

Comment:
nice tab ;)

Reply:
weird that doesn't show up in my editor :O

Reply:
oh never mind, restarted vscode and it was there